upbound/platform-ref-azure
# feat(backport): upgrade, cleanup before backport from caas

### Description of your changes
- upgrade build module
- bump uptest
- bump up-cli
- remove outdated pictures without reference to documents
- switch package to apis
- update makefile
- stop running publish action in forks without ci variable



Fixes #

I have:


### How has this code been tested
